.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: scala
+script:
+- "./bin/runci.sh $CI_TEST"
+jdk:
+- oraclejdk8
+env:
+  matrix:
+  - CI_TEST: scalafmt
+  - CI_TEST: ci-fast
```

Comment:
Is this defined in build.sbt? We can probably just run `test`

Reply:
The task is defined in `bin/runci.sh` It's currently failing because it's looking for `dragos-vscode-scala:0.2.1-SNAPSHOT`, but that's not a managed dependency in the `build.sbt` :O I'm confused...

Reply:
I'm very puzzled by this error, can you try `dragos-vscode-scala/publishLocal` before compile?